maintainers/maintainer-list.nix

```diff
@@ -4097,6 +4097,12 @@
     githubId = 131599;
     name = "Martin Weinelt";
   };
+  hexchen = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
alphabetic sorting seems to have broken during rebasing at some point, fixed it now.